### DISCUSSION TITLE
Multiple versions of Reanimated were detected. Only one instance of react-native-reanimated can be installed in a project. You need to resolve the conflict manually. Check out the documentation: https://docs.swmansion.com/react-native-reanimated/docs/fundamentals/troubleshooting#multiple-versions-of-reanimated-were-detected

### DISCUSSION THREAD

**Discussion by souna23** [software-mansion/react-native-reanimated#3636] [Q&A]
this problem happend after upgrading to version 2.10.0 but if i downgrade to version 2.9.1 everything is okey 

this solution not working too !!!! 

https://docs.swmansion.com/react-native-reanimated/docs/next/fundamentals/troubleshooting/

pls help

**Comment by tomekzaw** [comment]
Can you paste the output of `npm ls react-native-reanimated` here?

**Reply by souna23** [reply]
yes ofcourse

myproject@0.0.1 /Users/m/Documents/...path to my project
├─┬ @react-navigation/drawer@6.5.0
│ └── react-native-reanimated@2.10.0 deduped
└── react-native-reanimated@2.10.0
